[network_events.py]
  expected: TIMEOUT
